orchestrator/app.py

```diff
@@ -95,6 +99,7 @@ def _index() -> str:
 
     def instrument_app(self) -> None:
         logger.info("Activating Opentelemetry tracing to app", app=self.title)
+        self.base_settings.TRACING_ENABLED = True
```

Comment:
This feels wrong...

Reply:
It is wrong, good catch, this was some debug stuff.. Will change.